src/docs_website/pandoc/markdown-links.lua

```diff
@@ -1,6 +1,8 @@
 is_readme = PANDOC_STATE.input_files[1]:sub(-9) == "README.md"
 
 function Link (link)
+  local is_external = link.target:sub(1, 8) == "https://"
```

Comment:
In our link checker, once we do it, we should also verify that we never use http:// URLs